sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj

```diff
@@ -19,7 +19,7 @@
 
     <PackageReference Include="Azure.Core" />
     -->
-    <PackageReference Include="Azure.Core.Experimental" />
+    <!--    <PackageReference Include="Azure.Core.Experimental" />-->
```

Comment:
```suggestion
    <ProjectReference Include="..\..\..\core\Azure.Core.Experimental\src\Azure.Core.Experimental.csproj" />
```

Reply:
If we can, I'd appreciate keeping the change grouped to make it easier to understand what is related to which comment block.